docs/pages/setup/reference/metrics.mdx

```diff
@@ -109,6 +109,8 @@ Now you can see the monitoring information by visiting several endpoints:
 | `rx` | counter | Teleport | Number of bytes received. |
 | `server_interactive_sessions_total` | gauge | Teleport | Number of active sessions. |
 | `teleport_build_info` | gauge | Teleport | Provides build information of Teleport including gitref (git describe --long --tags), Go version, and Teleport version. The value of this gauge will always be 1. |
+| `teleport_cache_events` | counter | Teleport | Number of events received by a Teleport service cache. |
```

Comment:
I'm not sure if users who aren't familiar with the Teleport source will know what the "service cache" is since, as far as I know, it's not documented anywhere. Is there a way to expand on what the service cache is in this metric description? For example, "Teleport's Database Service, Kubernetes Service, and other services each cache incoming events."

Reply:
Updated help message and doc message, PTAL